package.json

```diff
@@ -45,8 +45,8 @@
     "ethjs-abi": "^0.2.1",
     "lodash.flatten": "^4.4.0",
     "semver": "^5.6.0",
-    "web3": "^1.2.1",
-    "web3-utils": "^1.2.1"
+    "web3": "^1.2.5",
+    "web3-utils": "^1.2.5"
```

Comment:
Are these dependency upgrades necessary for the feature?

Reply:
Yes.  See changelog https://github.com/ethereum/web3.js/blob/1.x/CHANGELOG.md#125